tests/unit/api/test_token.py

```diff
@@ -173,3 +173,40 @@ class User:
             "detail": {"error": "scope 'write:targets' forbidden"}
         }
         assert mocked_bcrypt.checkpw.calls == [pretend.call(b"test", "test")]
+
+    def test_post_with_empty_scope(self, monkeypatch, test_client):
```

Comment:
I don't quite understand how can I debug those tests locally from vscode?
How do you execute it?
Do I have always to run it with `pytest` to get this `monkeypatch` fixture?

Reply:
Yes, `monkeypatch` is a [Mock](https://docs.pytest.org/en/7.1.x/how-to/monkeypatch.html) fixture built-in in the `pytest`.
Not sure if I got the point here.

To the Unit Test I'm Mocking the backend generating the token, the `datetime`. To be honest, in this specific test case isn't required this Mock, once I don't assert "date time" anywhere.